gandalf.js

```diff
@@ -1,34 +1,33 @@
 /**
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?